src/coreclr/inc/corjit.h

```diff
@@ -404,24 +414,6 @@ class ICorJitInfo : public ICorDynamicInfo
             uint8_t **                pInstrumentationData         // OUT: `*pInstrumentationData` is set to the address of the instrumentation data.
             ) = 0;
 
-    // Get the likely implementing class for a virtual call or interface call made by ftnHnd
```

Comment:
Changing the Jit Interface requires an update to JITEEVersionIdentifier located in jiteeversionguid.h

Reply:
Oops, I thought I had done that.